Makefile

```diff
@@ -118,3 +119,16 @@ setup:
 	$(info $(INFOMARK) Installing Makefile go binary dependencies $(ROOTMARK) ...)
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
 	go install mvdan.cc/gofumpt@latest
+
+################################################################################
+# Target: version-docs                                                         #
+################################################################################
+.PHONY: version-docs
+version-docs:
+	$(info $(INFOMARK) Creating versioned docs ...)
+	docker run --rm \
+		-v $(shell pwd)/website:/website \
+		-w /website \
+		$(IDFLAGS) \
+		node:${NODE_VERSION} \
+		sh -c "yarn install --frozen lockfile && yarn run docusaurus docs:version ${CLI_VERSION}"
```

Comment:
Add missing newline ... eslint for go checks for this but not for general file types, and it's nice to have our source files play nice with POSIX tools that expect text files that end with newline.